packages/chakra-ui-docs/docs/tabs.mdx

```diff
@@ -1 +1,339 @@
-# Tabs
\ No newline at end of file
+import SEO from '../components/SEO'
+
+<SEO
+  title="Tabs"
+  description="The Tab component consists of clickable tabs, that are aligned side by side."
+/>
+
+# Tabs
+
+An accessible tabs component.
+
+The `Tab` and `TabPanel` elements are associated by their order in the tree.
+None of the components are empty wrappers, each is associated with a real DOM
+element in the document, giving you maximum control over styling and
+composition.
+
+## Import
+
+```js
+import { CTabs, CTabList, CTabPanels, CTab, CTabPanel } from '@chakra-ui/vue'
+```
+
+## Usage
+
+You can render any element within `CTabs`, but `CTabList` should only have `CTab`
+as children, and `CTabPanels` should have `CTabPanel` as children.
+
+Tabs expects `CTabList` and `CTabPanels` as children. The order doesn't matter,
+you can have tabs at the top, at the bottom, or both.
+
+```vue live=true
+<c-box w="600px">
+    <c-tabs>
+        <c-tab-list>
+        <c-tab>One</c-tab>
+        <c-tab>Two</c-tab>
+        <c-tab>Three</c-tab>
+        </c-tab-list>
+        <c-tab-panels>
+        <c-tab-panel>
+            <p>one!</p>
+        </c-tab-panel>
+        <c-tab-panel>
+            <p>two!</p>
+        </c-tab-panel>
+        <c-tab-panel>
+            <p>three!</p>
+        </c-tab-panel>
+        </c-tab-panels>
+    </c-tabs>
+</c-box>
```

Comment:
The code indentation here is different from how you normally write. So it's hard to read. Should be something like:

```vue
<c-box w="600px">
    <c-tabs>
      <c-tab-list>
        <c-tab>One</c-tab>
        <c-tab>Two</c-tab>
        <c-tab>Three</c-tab>
      </c-tab-list>

      <c-tab-panels>
        <c-tab-panel>
          <p>one!</p>
        </c-tab-panel>
        <c-tab-panel>
          <p>two!</p>
        </c-tab-panel>
        <c-tab-panel>
          <p>three!</p>
        </c-tab-panel>
      </c-tab-panels>
    </c-tabs>
  </c-box>
```

Reply:
I often find issue leaving spaces in the code blocks